include/ck/utility/math.hpp

```diff
@@ -152,21 +152,18 @@ __host__ __device__ constexpr T clamp(const T& x, const T& lowerbound, const T&
 
 // disallow implicit type casting
 template <typename T>
-__device__ T exp(T x);
-
-// TODO: add f16 support using v_exp_f16
-
-template <>
-__device__ float exp<float>(float x)
-{
-    return __expf(x);
```

Comment:
is ```__expf``` producing wrong result?

Reply:
I get correct results in AIT with gfx90a and I verify results with following example in CK.
``` cpp
int main(int argc, char* argv[])
{
    ck::half_t x_h = -1.231321;
    float x_h2 = -1.231321;
    ck::half_t y_h;
    float y_h2;
    ck::half_t* x;
    ck::half_t* y;
    hipMalloc(&x, sizeof(ck::half_t));
    hipMalloc(&y, sizeof(ck::half_t));
    hipMemcpy(x, &x_h, sizeof(ck::half_t), hipMemcpyHostToDevice);
    ck::tensor_operation::element_wise::Sigmoid{}(*y, *x);
    hipMemcpy(&y_h, y, sizeof(ck::half_t), hipMemcpyDeviceToHost);
    ck::tensor_operation::element_wise::Sigmoid{}(y_h2, x_h2);
    std::cout<<ck::type_convert<float>(y_h)<<' '<<ck::type_convert<float>(y_h2)<<std::endl;
}

```